go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/joschahenningsen/TUM-Live
 
-go 1.18
+go 1.19
```

Comment:
The version specified in go.mod is a constraint about the minimum go version required to run the app: https://go.dev/ref/mod#go-mod-file-go

In our case we could still allow running tum-live with go 1.18